.readthedocs.yml

```diff
@@ -0,0 +1,26 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+
+mkdocs:
+  configuration: mkdocs.yml
+
+# Optionally declare the Python requirements required to build your docs
+python:
+   install:
+   - method: pip
+     path: .
+     extra_requirements:
+       - docs
+
+formats:
+  - pdf
```

Comment:
are there pros and cos of setting pdf instead of html?

Reply:
`formats` specifies the additional formats (see [here](https://docs.readthedocs.io/en/stable/config-file/v2.html#formats)), so the docs will always be built as the html webpage. I included pdf and htmlzip, because then users can to download the docs as a single pdf file (might be easier for some folks to annotate) or a directory of html files (more faithfully represent the appearance of the website). I'm not sure anyone actually uses these, so I don't have strong feelings about including them, but thought it might be nice.